Donation-platform/src/main/webapp/trending.html

```diff
@@ -4,10 +4,18 @@
     <meta charset="UTF-8">
     <title>Trending Page</title>
     <script src="trendingScript.js"></script>
+    <link rel="stylesheet" href="trendingStyle.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
   </head>
+
   <body>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
We have to use jQuery for charity cards?

Reply:
no, I don't think so - I just added it because it was in one of the tutorials I was looking at.